dashboard/src2/components/BrandLogo.vue

```diff
@@ -0,0 +1,39 @@
+<template>
+	<span v-if="header">
+		<img
+			v-if="brandDetails && brandDetails.app_logo"
+			:src="brandDetails.app_logo"
+			class="inline-block h-7 w-7"
+		/>
+		<FCLogo class="inline-block h-7 w-7" v-else />
+	</span>
+
+	<span v-else-if="footer">
+		<img
+			v-if="brandDetails && brandDetails.app_footer_logo"
+			:src="brandDetails.app_footer_logo"
+		/>
+		<FrappeLogo class="h-4" v-else />
+	</span>
+
+	<span v-else> </span>
+</template>
+<script>
+import FCLogo from '@/components/icons/FCLogo.vue';
+import FrappeLogo from '@/components/icons/FrappeLogo.vue';
+import { getBrandInfo } from '../data/branding';
+
+export default {
+	name: 'BrandLogo',
+	components: {
+		FCLogo,
+		FrappeLogo
+	},
+	props: ['header', 'footer'],
```

Comment:
on 2nd thought this should've been a type prop with options being header and footer. Take a look at [AlertBanner.vue](https://github.com/frappe/press/blob/master/dashboard/src2/components/AlertBanner.vue) file

Then we can do
```
<BrandLogo type="header" />
```